generated/nidigital/nidigital/session.py

```diff
@@ -2694,7 +2694,11 @@ def __exit__(self, exc_type, exc_value, traceback):
     def initiate(self):
         '''initiate
 
-        TBD
+        Starts bursting the pattern, causing the NI-Digital session
```

Comment:
Change to "Starts bursting the pattern configured by NIDIGITAL_ATTR_START_LABEL, ..."

For details on the "NIDIGITAL_ATTR_" format, see the comment below in functions.py.